Multisig/UI/App/UIKit/AppSettingsViewController/AppSettingsViewController.swift

```diff
@@ -0,0 +1,226 @@
+//
+//  AppSettingsViewController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 10.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+fileprivate protocol SectionItem {}
+
+class AppSettingsViewController: UITableViewController {
+    private let app = App.configuration.app
+    private let legal = App.configuration.legal
+    private let tableBackgroundColor: UIColor = .gnoWhite
+    private let advancedSectionHeaderHeight: CGFloat = 28
+    var notificationCenter = NotificationCenter.default
+
+    private typealias SectionItems = (section: Section, items: [SectionItem])
+
+    private var sections = [SectionItems]()
+
+    enum Section {
+        case general
+        case advanced
+
+        enum General: SectionItem {
+            case importKey(String)
+            case importedKey(String, String)
+            case terms(String)
+            case privacyPolicy(String)
+            case licenses(String)
+            case getInTouch(String)
+            case appVersion(String, String)
+            case network(String, String)
+        }
+
+        enum Advanced: SectionItem {
+            case advanced(String)
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.backgroundColor = tableBackgroundColor
+        tableView.separatorStyle = .none
+        tableView.registerCell(BasicCell.self)
+        tableView.registerCell(ImportedKeyCell.self)
+        tableView.registerCell(InfoCell.self)
+        tableView.registerHeaderFooterView(BasicHeaderView.self)
+
+        buildSections()
+
+        notificationCenter.addObserver(
```

Comment:
I think for closing the controller having a closure in the `EnterSeedPhraseView` instead of the notification would be enough for this screen

Reply:
I'll remove in another PR